user_manual/conf.py

```diff
@@ -352,6 +358,19 @@ def setup(app):
     import laldoc
     app.add_directive('ada_auto_package', laldoc.AutoPackage)
 
+    import subprocess
+    from os import path as P
+    import sys
+
+    rst_content = subprocess.check_output([
+        sys.executable, P.join("..", "changes", "process_changes.py"),
+        "create-rst"
+    ])
+
+    with open("{}/api_changes.rst".format(P.dirname(P.abspath(__file__))),
```

Comment:
Using string formatting to create path just here while everything else uses `os.path` is odd: why not go all the way?

```
with open(P.join(P.dirname(P.abspath(__file__)), 'api_changes.rst'), 'w') as f:
```